riscv/sim.h

```diff
@@ -94,6 +94,10 @@ class sim_t : public htif_t, public simif_t
   void make_dtb();
   void set_rom();
 
+#ifdef RISCV_ENABLE_COMMITLOG
+  const char* get_symbol(uint64_t addr);
+#endif
```

Comment:
I don't think this should be necessary, since `get_symbol` is already defined on `htif_t`, which is a superclass.